lib/common/edna/server.nix

```diff
@@ -13,9 +13,7 @@
 
     virtualisation.docker = {
       enable = true;
-      logLevel = "warn";
       storageDriver = "overlay2";
-      networks.edna = {};
```

Comment:
Looks kinda breaking. [Module from serokell.nix](https://github.com/serokell/serokell.nix/blob/master/modules/virtualization/docker.nix) should probably be imported to preserve this network

Reply:
fixed